scripts/qa_crowbarsetup.sh

```diff
@@ -2790,6 +2801,14 @@ function custom_configuration()
                     provisioner_add_repo $repos "$tftpboot_repos12sp1_dir" "SUSE-OpenStack-Cloud-6-Updates-test" \
                         "http://$distsuse/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/"
                 fi
+
+                if iscloudver 7plus ; then
+                    repos="$autoyast['repos']['suse-12.1']"
+                    proposal_set_value provisioner default "$repos" "{}"
+
+                    provisioner_add_repo $repos "$tftpboot_repos12_dir" "SUSE-Enterprise-Storage-3-Updates-test" \
+                        "http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/3:/x86_64/update/"
+                fi
```

Comment:
This bit is not needed (and the previous block about 6plus can be removed) because this is inside a big `if ! iscloudver 6plus ; then`